src/java.base/share/classes/java/lang/Class.java

```diff
@@ -4674,17 +4674,12 @@ public boolean isSealed() {
      * The high order 16 bits contain the class's minor version.  The low order
      * 16 bits contain the class's major version.
      *
-     * If the class is an array type then the class file version of its component
+     * If the class is an array type then the class file version of its element
      * type is returned.  If the class is a primitive type then the latest class
      * file major version is returned and zero is returned for the minor version.
      */
     private int getClassFileVersion() {
-        Class<?> component = this;
-        if (isArray()) {
-            do {
-                component = component.getComponentType();
-            } while (component.isArray());
-        }
+        Class<?> component = isArray() ? elementType() : this;
```

Comment:
The variable name `component` seems completely not applicable now. I would just use `c`.

Reply:
Done.